main/src/cgeo/geocaching/log/LoggingUI.java

```diff
@@ -93,6 +95,9 @@ private static void showOfflineMenu(final Geocache cache, final Activity activit
             list.add(new LogTypeEntry(null, SpecialLogType.CLEAR_LOG, false));
         }
         list.add(new LogTypeEntry(null, SpecialLogType.LOG_CACHE, false));
+        if (Settings.getLogTemplates().size() > 0 && (logTypes.contains(LogType.FOUND_IT) || logTypes.contains(LogType.WEBCAM_PHOTO_TAKEN))) {
```

Comment:
do we need to take `LogType.ATTENDED` into consideration as well?

Reply:
I decided against it, even thought about not having it for webcam but then included it as on a cache tour including a webcam you might log the webcam the same as the caches - unlikely for the event.

Reply:
Agreed, but you might define different default templates for different cache types. (This was actually requested recently on support, paired with "use it automatically", but that's a different story.)

Reply:
There's no association between log types and templates. They are just text.

Reply:
I know (which would rule out any automatism), still the user could create their "Found it (Webcam)", "Found it (Event)" and "Found it (everything else)" default log templates (in whatever combination).

But anyway, no blocker for me. We can merge it and see whatever user feedback may come in.